app/views/comments/edit.js.erb

```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render "comments/form", comment: @comment) %>");
```

Comment:
This is really good! Great job identifying which template to render for edit vs update!